### DISCUSSION TITLE
abc  WABC-DT1  (us)

### DISCUSSION THREAD

**Discussion by erich1997** [iptv-org/community#955] [Q&A]
### Channel ID

WABCDT1.us

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://github.com/iptv-org/iptv/blob/2b5c25f1dcd5c0c7a689dff74e407da37926eae3/streams/us_local.m3u#L546-L547 ?

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_local.m3u#L34-L35
